Automattic/wp-calypso
# Stats: Add missing hyphens in upsell notice and modal

## Proposed Changes

Usage is adjective describing a noun so "scaling up" and "money back" should be compound forms with a hyphen.

<img width="758" alt="SCR-20240105-nfpi" src="https://github.com/Automattic/wp-calypso/assets/40267301/92812c6e-ac77-46a3-89d5-ad20ae3387d5">

## Testing Instructions



- Visit Stats page for a DotCom site.
- Confirm the upsell notice has the "scaling-up" fix.
- Click the Upgrade button.
- Confirm the modal has both fixes per above screenshot.

## Pre-merge Checklist
